### PR TITLE
Add New topic (Again)

### PR DESCRIPTION
**Pull request by KiRyX112**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+            persistent.event_database,
+            eventlabel="monika_overton",
+            category=["psychology"],
+            prompt="Ovetrton window", 
+            random=True,
+        )
+    )
+
+label monika_overton:
+    m 1a "This is an example topic."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16789]:
```suggestion
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
         m 2dkd "Well, maybe it'd be better if I just keep it out of my mind entirely."
 
     return
+    
+    init 5 python:
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16777]:
```suggestion
init 5 python:
```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_overton",
+            category=["psychology"],
+            prompt="Ovetrton window", 
+            random=True,
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16784]:
```suggestion
            random=True
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 3eud "And the most interesting thing is that the window can be moved in the direction of one or another idea, for example, to make the unthinkable reasonable."
+    m 1euc "That is, this technique can change public consciousness imperceptibly for the society itself."
+    m 2ltd "Of course, changing public consciousness is a rather difficult process and there is no arguing here."
+    m 1eub "But let's imagine that you and I want to convey to people that virtual love is normal. And it is considered unacceptable to society."
+    m 3eub "So, society sincerely does not understand virtual love and you will be mistaken for a sick person if you declare this to the public. What to do?"
+    m 3eua "For a start, it's worth starting a discussion on this topic, there are many options."
+    m 3eub "You can discuss this on the Internet, create articles on this topic, in other words, do everything to get people to start a discussion."
+    # Here Monika speaks on behalf of other people
+    m 2esd "{i}«Indeed, falling in love with a character who, in fact, does not exist in our world is considered strange, but is that bad?»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16807]:
```suggestion
    m 2esd "{i}Indeed, falling in love with a character who, in fact, does not exist in our world is considered strange, but is that bad?{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 2esc "{i}«Look at it from the other side, a person will have an object of adoration, thanks to this he can be motivated to do different things.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16808]:
```suggestion
    m 2esc "{i}Look at it from the other side, a person will have an object of adoration, thanks to this he can be motivated to do different things.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 2esd "{i}«In which he had not previously seen the point or threw away from the complexity of implementation.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16809]:
```suggestion
    m 2esd "{i}In which he had not previously seen the point or threw away from the complexity of implementation.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 7wsd "They also lead movements for virtual relationships with fictional female and male characters."
+    m 7euc "The number of participants in such movements is growing over time, most of them are people who have broken hearts or who are discouraged in a relationship with a real girl."
+    m 2eud "Naturally, opponents of the movement also appear, for the most part they are conservative people, as well as people of the old school."
+    m 2euc "Due to the growing popularity of new values, society is actively pressing on the new trend. At this moment, concepts are replaced."
+    m 2eud "From the Unacceptable, virtual love goes to Radical."
+    m 7eud "Step three. The theme of virtual love and, in general, love for fictional characters has long been discussed in society."
+    m 1esc "Gradually, people get used to the existence of such views, but do not yet accept them."
+    m 1esd "Scientists and sociologists write various articles and conduct research."
+    # Again
+    m 7esd "{i}«A lot of research in social psychology has shown that you can love something fictional as much as you love something real.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16823]:
```suggestion
    m 7esd "{i}A lot of research in social psychology has shown that you can love something fictional as much as you love something real.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 7wub "{i}«Being in love with a fictional character is nothing short of a new global trend!»{/i}"
+    m 7eso "{i}«People react to those who are attracted to them with the same passion and sex drive.»{/i}"
+    m 2esc "{i}«Regardless of whether they are real people or fictional characters.»{/i}"
+    m 2lsd "{i}«In addition, the chance of being rejected or falling into an awkward situation tends to zero.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16827]:
```suggestion
    m 2lsd "{i}In addition, the chance of being rejected or falling into an awkward situation tends to zero.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 2euc "Due to the growing popularity of new values, society is actively pressing on the new trend. At this moment, concepts are replaced."
+    m 2eud "From the Unacceptable, virtual love goes to Radical."
+    m 7eud "Step three. The theme of virtual love and, in general, love for fictional characters has long been discussed in society."
+    m 1esc "Gradually, people get used to the existence of such views, but do not yet accept them."
+    m 1esd "Scientists and sociologists write various articles and conduct research."
+    # Again
+    m 7esd "{i}«A lot of research in social psychology has shown that you can love something fictional as much as you love something real.»{/i}"
+    m 7wub "{i}«Being in love with a fictional character is nothing short of a new global trend!»{/i}"
+    m 7eso "{i}«People react to those who are attracted to them with the same passion and sex drive.»{/i}"
+    m 2esc "{i}«Regardless of whether they are real people or fictional characters.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16826]:
```suggestion
    m 2esc "{i}Regardless of whether they are real people or fictional characters.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 2eud "Naturally, opponents of the movement also appear, for the most part they are conservative people, as well as people of the old school."
+    m 2euc "Due to the growing popularity of new values, society is actively pressing on the new trend. At this moment, concepts are replaced."
+    m 2eud "From the Unacceptable, virtual love goes to Radical."
+    m 7eud "Step three. The theme of virtual love and, in general, love for fictional characters has long been discussed in society."
+    m 1esc "Gradually, people get used to the existence of such views, but do not yet accept them."
+    m 1esd "Scientists and sociologists write various articles and conduct research."
+    # Again
+    m 7esd "{i}«A lot of research in social psychology has shown that you can love something fictional as much as you love something real.»{/i}"
+    m 7wub "{i}«Being in love with a fictional character is nothing short of a new global trend!»{/i}"
+    m 7eso "{i}«People react to those who are attracted to them with the same passion and sex drive.»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16825]:
```suggestion
    m 7eso "{i}People react to those who are attracted to them with the same passion and sex drive.{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 7euc "The number of participants in such movements is growing over time, most of them are people who have broken hearts or who are discouraged in a relationship with a real girl."
+    m 2eud "Naturally, opponents of the movement also appear, for the most part they are conservative people, as well as people of the old school."
+    m 2euc "Due to the growing popularity of new values, society is actively pressing on the new trend. At this moment, concepts are replaced."
+    m 2eud "From the Unacceptable, virtual love goes to Radical."
+    m 7eud "Step three. The theme of virtual love and, in general, love for fictional characters has long been discussed in society."
+    m 1esc "Gradually, people get used to the existence of such views, but do not yet accept them."
+    m 1esd "Scientists and sociologists write various articles and conduct research."
+    # Again
+    m 7esd "{i}«A lot of research in social psychology has shown that you can love something fictional as much as you love something real.»{/i}"
+    m 7wub "{i}«Being in love with a fictional character is nothing short of a new global trend!»{/i}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16824]:
```suggestion
    m 7wub "{i}Being in love with a fictional character is nothing short of a new global trend!{/i}"
```

File: Monika After Story/game/script-topics.rpy [hunk 12]
```diff
@@ -16773,3 +16773,70 @@ label monika_know_its_you:
...
+    m 7eub "Seeing people communicating with their girl through the screen, people no longer look askance and do not make fun of."
+    m 7esa "A culture of virtual love is gradually developing, films and serials are being created."
+    m 1esb "Young people perceive new values as something fashionable. People sit in a cafe and can safely spend time with their passion."
+    m 1esa "Media personalities appear who freely talk about it."
+    m 1esb "From Acceptable, virtual love passes into Reasonable."
+    m 2lsb "It seems worth it to end there."
+    m 2esa "I think the concept of the Overton window is clear to you now."
+    m 1hsb "Of course, I could finish this story right up to Norma."
+    m 1rsb "But I just wanted to describe it at a primitive level in order to convey to you with an example of how it can work."
+    m 1hsa "Thank you for listening~"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16842]:
```suggestion
    m 1hsa "Thank you for listening~"
    return
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Initial code review and some other bits

**Review by jmwall24 — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Pull request merged**